lib/membrane_portaudio.ex

```diff
@@ -20,6 +20,8 @@ defmodule Membrane.PortAudio do
   @spec print_devices() :: :ok
   def print_devices() do
     Application.ensure_all_started(:membrane_portaudio_plugin)
+
     __MODULE__.SyncExecutor.apply(__MODULE__.Devices, :list, [])
+    |> IO.inspect()
```

Comment:
Let's add `list_devices/0` here too, the existing one is in a private module and not guarded with the sync executor